recipes/functions-framework-cpp/all/conanfile.py

```diff
@@ -0,0 +1,97 @@
+import os
+
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.36.0"
+
+
+class FunctionsFrameworkCppConan(ConanFile):
+    name = "functions-framework-cpp"
+    description = "An open source FaaS (Functions as a Service) framework"
+    license = "Apache-2.0"
+    topics = "google", "cloud", "functions-as-a-service", "faas-framework"
+    homepage = "https://github.com/GoogleCloudPlatform/functions-framework-cpp"
+    url = "https://github.com/conan-io/conan-center-index"
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake", "cmake_find_package_multi", "cmake_find_package"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+
+    short_paths = True
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def validate(self):
+        tools.check_min_cppstd(self, "17")
+
+        if self.settings.os == "Windows" and self.options.shared:
+            raise ConanInvalidConfiguration("Fails to compile for Windows as a DLL")
+
+        if hasattr(self, "settings_build") and tools.cross_building(self):
+            raise ConanInvalidConfiguration(
+                "Recipe not prepared for cross-building (yet)"
+            )
+
+    def source(self):
+        tools.get(
+            **self.conan_data["sources"][self.version],
+            destination=self._source_subfolder,
+            strip_root=True
+        )
+
+    def requirements(self):
+        self.requires("abseil/20210324.2")
+        self.requires("boost/1.75.0")
+        self.requires("nlohmann_json/3.10.2")
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.definitions["BUILD_TESTING"] = 0
```

Comment:
```suggestion
        self._cmake.definitions["BUILD_TESTING"] = False
        self._cmake.definitions["FUNCTIONS_FRAMEWORK_CPP_TEST_EXAMPLES"] = False
```

don't enable tests and example: https://github.com/GoogleCloudPlatform/functions-framework-cpp/blob/v1.0.0/CMakeLists.txt#L27